content/examples/shortcodes/tag/en.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Tag
+weight: 33
+---
+
+To use tags inline use the `{{</* tag */>}}` shortcode. Just add the tag name to the shortcode. Use quotes around the tag name if it contains a space.
```

Comment:
Mention that only tags that are in the site params work. So if want you to add a tag which is not there yet you first have to it to the params